.github/workflows/publish.yml

```
@@ -0,0 +1,29 @@
+name: Publish Python Package
+
+on:
+  push:
+    branches: 
+      - master
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
+    - name: Build package
+      run: python -m build
+    - name: Publish package
+      uses: pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29
```

```suggestion
      uses: pypa/gh-action-pypi-publish@v1
```

as indicated in [usage](https://github.com/pypa/gh-action-pypi-publish#usage) section.